Scripts/sort-schemes.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Sort the order of projects' schemes.
+# Usage: sort-schemes.sh
+
+# Alias for replacing symbols.
+aliasNewLine='@'
+aliasNumber='%numberAlias%'
+
+# Regex constans for change number.
+regexStartPlaceNumber="<key>[a-zA-z -^#.]*<\/key>[${aliasNewLine}][[:space:]]*<dict>[${aliasNewLine}][[:space:]]*<key>orderHint<\/key>[${aliasNewLine}][[:space:]]*<integer>"
+regexEndPlaceNumber="<\/integer>[${aliasNewLine}][[:space:]]*<\/dict>"
+
+# Regex constans for hidden schemes.
+regexHiddenScheme="<key>isShown<\/key><false\/>"
+regexStartHiddenScheme="<key>[a-zA-z -^#.]*<\/key>[${aliasNewLine}][[:space:]]*<dict>[${aliasNewLine}][[:space:]]*"
+regexEndHiddenScheme="<key>orderHint<\/key>[${aliasNewLine}][[:space:]]*<integer>[0-9]*<\/integer>[${aliasNewLine}][[:space:]]*<\/dict>"
+
+# Scheme counter.
+currentSchemeNumber=0;
+
+replace_number_or_hide_scheme() {
+
+    # Get params.
+    projectName=$1
+    isHidden=$2
+
+    # Build path to file of scheme.
+    fileName="${projectName}/xcuserdata/$USER.xcuserdatad/xcschemes/xcschememanagement.plist"
+
+    # Prepare the backup file.
+    backupFile="${fileName}.bak"
```

Comment:
It's just a var, why "prepare"?
```suggestion
    # Backup file name.
    local backupFile="${fileName}.bak"
```